sdk/scheduler/src/main/java/com/mesosphere/sdk/specification/DefaultService.java

```diff
@@ -96,12 +103,30 @@ private void initService() {
                 StateStoreUtils.setUninstalling(stateStore);
             }
 
+            Optional<SecretsClient> secretsClient = Optional.empty();
+            try {
+                TokenProvider tokenProvider = TLSEvaluationStage.Builder.tokenProviderFromEnvironment(
+                        schedulerBuilder.getSchedulerFlags());
+                Executor executor = Executor.newInstance(
+                        new DcosHttpClientBuilder()
+                                .setTokenProvider(tokenProvider)
+                                .setRedirectStrategy(new LaxRedirectStrategy())
+                                .build());
+                secretsClient = Optional.of(new DefaultSecretsClient(executor));
+            } catch (Exception e) {
+                LOGGER.error("Failed to create a secrets store client," +
```

Comment:
I've added following code:

```
            Optional<SecretsClient> secretsClient = Optional.empty();
            if (!TaskUtils.getTasksWithTLS(getServiceSpec()).isEmpty()) {
```

Having config validator and this condition the error we'd run TLS cleanup block only if any of tasks is configured with TLS. This should cover not displaying any message on DC/OS Open.